app/controllers/concerns/europeana/entities.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Europeana
+  module Entities
+    extend ActiveSupport::Concern
+    include EntitiesHelper
+
+    protected
+
+    def build_query_items_by(params)
```

Comment:
Do we need an Entities concern? are you planing on using build_query_items_by somewhere other than the entities_controller?